src/Sylius/Bundle/InstallerBundle/Command/SetupCommand.php

```diff
@@ -108,7 +108,7 @@ protected function setupAdministratorUser(InputInterface $input, OutputInterface
             } while ($exists);
 
             $user->setEmail($email);
-            $user->setPlainPassword($this->askHidden($output, 'Choose password:', array(new NotBlank())));
+            $user->setPlainPassword($this->getAdministratorPassword($output));
```

Comment:
Sure, you're right. I have only one problem with using Symfony Dialog Helper - using it, we could only ask and validate password confirmation. What if **the first** password was typed wrongly? We could obviously nest one validator in another, but it smells really bad.